skipper.go

```diff
@@ -1365,11 +1371,49 @@ func run(o Options, sig chan os.Signal, idleConnsCH chan struct{}) error {
 
 	var swarmer ratelimit.Swarmer
 	var redisOptions *skpnet.RedisOptions
+	log.Infof("enable swarm: %v", o.EnableSwarm)
 	if o.EnableSwarm {
 		if len(o.SwarmRedisURLs) > 0 {
```

Comment:
For dynamic updates `SwarmRedisURLs` would be empty and the initial list should be discovered automatically I guess

Reply:
fixed